dogether/Presentation/Common/DogetherButton.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  DogetherButton.swift
+//  dogether
+//
+//  Created by seungyooooong on 2/9/25.
+//
+
+import Foundation
+import UIKit
+
+class DogetherButton: UIButton {
+    var action: () -> Void
```

Comment:
안녕하세요! 지나가다가 잠깐 들렀습니다!

```suggestion

final class DogetherButton: UIButton {
    var action: () -> Void
```
상속을 하지 않는 class에 final을 붙여주시면 좋을 거 같아요!